code/modules/mob/living/carbon/human/death.dm

```diff
@@ -131,4 +131,17 @@
 /mob/living/carbon/human/proc/vr_disconnect()
 	if(remote_network)
 		SSvirtualreality.remove_robot(src, remote_network)
-		remote_network = null
\ No newline at end of file
+		remote_network = null
+
+/mob/living/carbon/human/proc/drop_all_limbs(var/skip_qdel, var/droplimb_type = DROPLIMB_BLUNT)
```

Comment:
`skip_qdel` is only relevant to the `physically_destroyed()` proc this was copied from, you can baleet it here.